Lens/Resolver/ReferencedAssemblyCache.cs

```diff
@@ -77,6 +78,15 @@ public void ReferenceAssembly(Assembly asm)
             _assemblies.Add(asm);
         }
 
+        /// <summary>
+        /// Returns the loaded assemblies.
+        /// </summary>
+        private IEnumerable<Assembly> GetLoadedAssemblies()
+        {
+            var asms =  AppDomain.CurrentDomain.GetAssemblies();
+            return asms.Where(x => !x.FullName.StartsWith("System.Private."));
```

Comment:
Honestly looks like a bit of hack. Probably we need more explanation here at least. What bad consequences will we have if we return `System.Private.Smth` from this method?

Reply:
Moved to #196.